lib/helpers/normalizeArgs.js

```diff
@@ -16,7 +16,10 @@ function normalizeArgs(registry, args) {
     return fn;
   }
 
-  return map(flatten(args), getFunction);
+  var flattenArgs = flatten(args);
+  assert(flattenArgs.length, 'Arguments must be a non-empty array or string');
```

Comment:
I think this message should be more user friendly.  How does "One or more tasks must be combined using series or parallel" sound?